interpreter/CMakeLists.txt

```diff
@@ -52,9 +52,10 @@ set(LLVM_EXTERNAL_PROJECTS "cling" CACHE STRING "")
 # to serialize access to llvm.  We can later review how to make this finer grained by using llvm's own locking
 # mechanism.
 set(LLVM_ENABLE_THREADS OFF CACHE BOOL "")
-if(cxx17)
+
+if(CMAKE_CXX_STANDARD EQUAL 17)
   set(LLVM_ENABLE_CXX1Z ON CACHE BOOL "" FORCE)
-elseif(cxx14)
+elseif(CMAKE_CXX_STANDARD EQUAL 14)
   set(LLVM_ENABLE_CXX1Y ON CACHE BOOL "" FORCE)
```

Comment:
That looks surprising to me - why don't we turn on `LLVM_ENABLE_CXX14`? And yes, that should probably not be part of this PR.

Reply:
I don't know, I just thought that `LLVM_ENABLE_CXX1Y` is the only way to tell it to enable C++14...

Reply:
Just for future reference:
```
epsft-53 root $ git grep LLVM_ENABLE_CXX
interpreter/CMakeLists.txt:  set(LLVM_ENABLE_CXX1Z ON CACHE BOOL "" FORCE)
interpreter/CMakeLists.txt:  set(LLVM_ENABLE_CXX1Y ON CACHE BOOL "" FORCE)
interpreter/llvm/src/CMakeLists.txt:option(LLVM_ENABLE_CXX1Y "Compile with C++1y enabled." OFF)
interpreter/llvm/src/CMakeLists.txt:option(LLVM_ENABLE_CXX1Z "Compile with C++1z enabled." OFF)
interpreter/llvm/src/cmake/modules/HandleLLVMOptions.cmake:  if (LLVM_ENABLE_CXX1Y)
interpreter/llvm/src/cmake/modules/HandleLLVMOptions.cmake:  elseif(LLVM_ENABLE_CXX1Z)
interpreter/llvm/src/docs/CMake.rst:**LLVM_ENABLE_CXX1Y**:BOOL
```